src/lib/functions/netlify-function.js

```diff
@@ -41,7 +41,7 @@ class NetlifyFunction {
     // Determines whether this is a background function based on the function
     // name.
     this.isBackground = name.endsWith(BACKGROUND_SUFFIX)
-    this.schedule = null
+    this.schedule = config.functions[name]?.schedule
```

Comment:
FYI we don't support the `?.` operator with the current Node target.

Reply:
Yeah, I noticed sadly.